tekton/tasks/run-tests.yaml

```diff
@@ -94,4 +94,5 @@ spec:
 
         echo "running tests"
         IntegrationServer --work-dir $TEST_SERVER \
-          --test-project $(params.test-project-name)
+          --test-project $(params.test-project-name) \
+          --start-msgflows false 
```

Comment:
whoops 🤦‍♂️

I'd even copied the command from https://www.ibm.com/docs/en/app-connect/12.0?topic=tests-running-by-using-command-line so I don't know how I missed this :)